draft-welzl-ccwg-ratelimited-increase.md

```diff
@@ -122,13 +122,15 @@ For example, for Slow Start, as specified in {{!RFC5681}}, limit(maxFS)=2*maxFS,
 cwnd_new = cwnd + min (N, SMSS)
 cwnd = min(cwnd_new, 2*maxFS)
 ~~~
+where cwnd and SMSS follow their definitions in {{!RFC5681}} and N is the number of previously unacknowledged bytes acknowledged in the incoming ACK.
```

Comment:
```suggestion
where cwnd and SMSS follow their definitions in {{!RFC5681}}, which states that N "is the number of previously unacknowledged bytes acknowledged in the incoming ACK".
```

Reply:
The point of this is to clarify that this is the original wording from RFC 5681. I don't think we should attempt to massage that text instead, since it's copy+pasted from there.